app/views/directives/lifecycle-hook.html

```diff
@@ -115,13 +96,14 @@
               <label class="required">Tag As</label>
               <istag-select model="istagHook" allow-custom-tag="true" select-disabled="view.isDisabled"></istag-select>
             </div>
-            <p ng-if="hookParams.tagImages.length > 1">
+          </div>
+
+          <div class="read-only-tag-image" ng-if="hookParams.tagImages.length > 1">
+            <p class="read-only-info" ng-if="$first">
               <span class="pficon pficon-info" aria-hidden="true"></span>
-              More then one image tags are defined in this deployment configuration. Only the first one is editable, below is displayed the rest.
+              More then one image tags are defined in this deployment configuration. To edit them, use deployment config YAML editor.
```

Comment:
"More than one image tag is defined. To change image tags, use the YAML editor."